### PR TITLE
Feature/tic tac toe

### PR DESCRIPTION
**Pull request by damiecode**
Thank you for taking the time to review our project. Work-done
- [x] Created an interface to play the game on the browser
- [x] Created the game logic using GameBoard, Player and DisplayController Object.
- [x] Ensured player cannot play twice on the same spot
- [x] Displayed player scores.

### CODE REVIEW

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi Team :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just one more required to work on............ :slightly_smiling_face: 
##### Let make your repo a professional one.
README file not detailed enough. (you can add steps to git clone your repo, necessary installation and live link)
Kindly watch the video tutorial on how to create live link below 👇;
https://www.loom.com/share/76088842a2d14322bbcf65b884f4d42d
checkout out the below link for help, free free to follow the template from the pathwright
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/

**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Comment by damiecode** [comment]
Thank you @bellom, We have updated the README file.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,47 @@
...
+[OdinProject](https://www.theodinproject.com/courses/javascript/lessons/tic-tac-toe-javascript)
+
+## Built With
+
+- HTML/CSS,
+- Javascript
+
+## Live Demo
+
+[Live Demo Link](https://rawcdn.githack.com/damiecode/Tic-Tac-Toe-Js/feature/Tic-Tac-Toe/index.html )
```
Comment (bolah2009) [inline README.md]: The live link may become broken if the `feature` branch is deleted.

You could use `https://rawcdn.githack.com/damiecode/Tic-Tac-Toe-Js/feature/Tic-Tac-Toe/index.html` instead or a commit specific URL like `https://rawcdn.githack.com/damiecode/Tic-Tac-Toe-Js/ae91a6112a67103254fe556cf2b11d5712f1b1b1/index.html`

**Review by bolah2009 — COMMENTED** [review]
Awesome work with the [app](https://rawcdn.githack.com/damiecode/Tic-Tac-Toe-Js/ae91a6112a67103254fe556cf2b11d5712f1b1b1/index.html)

It's a 👍🏾 from me!

### :heavy_check_mark: Approved!

Check out the optional comments before merging. :relaxed: 

Good luck with your next projects. :star2: 

Cheers
[Bola](https://bolabuari.com/)

File: README.md (hunk 1 above)
Comment (damiecode) [reply README.md]: Thank you @bolah2009

**Pull request merged**